src/crypto/crypto_common.cc

```diff
@@ -469,13 +466,13 @@ MaybeLocal<Value> GetExponentString(
     Environment* env,
     const BIOPointer& bio,
     const BIGNUM* e) {
-  uint64_t exponent_word = static_cast<uint64_t>(BN_get_word(e));
+  uint64_t exponent_word = static_cast<uint64_t>(BignumPointer::getWord(e));
```

Comment:
Should we change the function name styling and start static functions with capital letters like V8?

Reply:
Probably a good idea.